src/main/java/roomescape/controller/ReservationTimeController.java

```diff
@@ -0,0 +1,62 @@
+package roomescape.controller;
+
+import java.net.URI;
+import java.time.LocalDate;
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.dto.AvailableReservationTimeResponse;
+import roomescape.dto.ReservationTimeRequest;
+import roomescape.dto.ReservationTimeResponse;
+import roomescape.service.ReservationTimeService;
+
+@RestController
+public class ReservationTimeController {
+
+    private final ReservationTimeService reservationTimeService;
+
+    public ReservationTimeController(ReservationTimeService reservationTimeService) {
+        this.reservationTimeService = reservationTimeService;
+    }
+
+    @GetMapping("/times")
+    public ResponseEntity<List<ReservationTimeResponse>> getAllReservationTimes() {
+        List<ReservationTimeResponse> reservationTimeResponses = reservationTimeService.getAllReservationTimes();
+
+        return ResponseEntity.ok(reservationTimeResponses);
+    }
+
+    @PostMapping("/times")
+    public ResponseEntity<ReservationTimeResponse> addReservationTime(
+            @RequestBody ReservationTimeRequest reservationTimeRequest) {
```

Comment:
```suggestion
    public ResponseEntity<ReservationTimeResponse> addReservationTime(
            @RequestBody ReservationTimeRequest reservationTimeRequest
     ) {
```

53 ~ 56 라인을 보면 이렇게 사용하고 있네요!
둘중 하나의 컨벤션을 가지고 통일하면 좋을것 같습니다!

Reply:
후자의 방법으로 컨벤션을 통일했습니다!